src/server/game/Entities/Player/Player.h

```diff
@@ -2158,6 +2158,8 @@ class Player : public Unit, public GridObject<Player>
     void UpdateSkillsToMaxSkillsForLevel();             // for .levelup
     void ModifySkillBonus(uint32 skillid, int32 val, bool talent);
 
+    void RewardOtherBonusTalentPoints(uint32 bonusTalentPoints);
```

Comment:
it would be nice to have a comment here explaining what is this method supposed to be used for, together with some guidance

cc @BarbzYHOOL

Reply:
@FrancescoBorzi please chack the comment. thx